ingestion_tools/scripts/importers/dataset_key_photo.py

```diff
@@ -16,12 +16,14 @@
 
 class DatasetKeyPhotoImporter(BaseImporter):
     type_key = "dataset_keyphoto"
+    plural_key = "dataset_keyphotos"
     image_keys = ["snapshot", "thumbnail"]
+    finder_factory = DefaultImporterFactory
```

Comment:
Does this handle cases where no link or sources are specified for `dataset_keyphotos` in configs?

Reply:
hehhh... actually it doesn't. Fixing!